web/src/components/metrics/histogram.tsx

```diff
@@ -241,7 +241,9 @@ export const Histogram: React.FC<{
       onKeyDown={e => onKeyDown(e.key)}
     >
       <Flex className="histogram-range-container" direction={{ default: 'row' }}>
-        <FlexItem flex={{ default: 'flex_1' }} />
+        <FlexItem flex={{ default: 'flex_1' }}>
+          <Text>{t('Number of logs over time')}</Text>
+        </FlexItem>
```

Comment:
Is there a value adding this title on top left corner of the histogram for you guys ?
It would replace a tooltip while hovering the graph